js/index.js

```diff
@@ -2,6 +2,22 @@ console.clear();
 
 const url = "https://swapi.dev/api/people";
 
+async function fetchData() {
+  try {
+    const response = await fetch(url);
+    // console.log(response);
+    // console.log(typeof response);
+    const data = await response.json();
+    // console.log(data);
+    // console.log(typeof data);
 
+    const foundItem = data.results.find((element) => element.name === "R2-D2");
+    // console.log(foundItem)
+    const eyeColor = foundItem.eye_color;
```

Comment:
can you access the eyecolor with dot notation? I would go with foundItem["eye_color"] because of the underscore. let me know if yours works.